api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/fields/format.md

```diff
@@ -162,6 +162,207 @@ See the [format](/api-reference/50%20Common/Object%20Structures/format '/Documen
 
 ---
 
+Date formats require additional configuration. The **PivotGridDataSource** groups date field values by date component: Year-Quarter-Month. For this, the values are converted from dates to numbers, but numbers cannot be formatted using date formats. To apply a date format in this case, you need to override this grouping.
+
+The following code declares a custom group for the `ShippingDate` data field. This configuration overrides this field's default grouping. Field values are grouped first by year, then by month. Months are formatted using the `MMM` format (`Jan`, `Feb`, `Mar`, etc.). This format is applied to the Date values returned from the [selector](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#selector) function.
```

Comment:
A possible option: Months are formatted using -> Month are formatted with the `MMM` format.

Example from the Microsoft docs: [By default, all worksheet cells are formatted with the General number format](https://docs.microsoft.com/en-us/office/troubleshoot/excel/format-cells-settings).